ProcessMaker/Http/Middleware/GenerateMenus.php

```diff
@@ -165,7 +165,7 @@ public function handle(Request $request, Closure $next)
                 'target' => '_blank'
             ]);
             $submenu->add(__('Report an issue'), [
```

Comment:
Recommend changing this to "Report Issue" to match the formal tone of the other menu items.